api/src/openapi/apiSchema.yml

```diff
@@ -295,7 +325,7 @@ paths:
         200:
           description: Successful response
 
-  /api/staticLayers:
+  /api/dataLayers:
```

Comment:
Hi, we still use word static in many place in this config regarding this endpoint. Should this be renamed also?

Reply:
yes, it should :) I just considered it out of scope here

Reply:
:) Sure. I fixed it on FE for layers to work.